### PR TITLE
RPC: update listkeys

### PR DESCRIPTION
**Pull request by MarnixCroes**
also returns _TaprootAccountKeyPath_ and address

### CODE REVIEW

File: docs/using-wasabi/RPC.md [hunk 1]
```diff
@@ -474,31 +474,44 @@ curl --data-binary '{"jsonrpc":"2.0","id":"1","method":"listkeys"}' http://127.0
...
+      "fullKeyPath": "86'/1'/0'/1/0",
       "internal": true,
-      "keyState": 1,
-      "label": "Bernardo",
-      "p2wpkhScript": "0 7242d77e2da9d3474190e465ed50f6b30b3cf2e9",
-      "pubkey": "03f31132f80ebdf480ad2aaf15631e9b643734ab78701012ba621b113c23ecab26",
-      "pubKeyHash": "7242d77e2da9d3474190e465ed50f6b30b3cf2e9"
+      "keyState": 0,
+      "label": "",
+      "p2wpkhScript": "0 6fdd4e1578612eaa6c3cac493244fc3536ae1169",
```
Comment (lontivero) [inline docs/using-wasabi/RPC.md:501]: Missleading. This key has a taproot derivation path but displays a p2wpkhScript and a wrong address. It seems we have to work a bit more on the RPC before udpating these examples.

File: docs/using-wasabi/RPC.md [hunk 2]
```diff
@@ -474,31 +474,44 @@ curl --data-binary '{"jsonrpc":"2.0","id":"1","method":"listkeys"}' http://127.0
...
+      "pubkey": "031606b4c6a8651726020af23cbd014c39695171444effd799ba00c41e2b37aad6",
+      "pubKeyHash": "9d71708d966c8900e40f5563aab93d5394c2cca5",
+      "address": "tb1qn4chprvkdjyspeq024364wfa2w2v9n99ucent4"
     },
     {
-      "fullKeyPath": "84'/0'/0'/0/2",
+      "fullKeyPath": "84'/1'/0'/0/0",
       "internal": false,
-      "keyState": 1,
-      "label": "Valeria",
```
Comment (lontivero) [inline docs/using-wasabi/RPC.md:489]: Do not remove the labels.

File: docs/using-wasabi/RPC.md [hunk 3]
```diff
@@ -474,31 +474,44 @@ curl --data-binary '{"jsonrpc":"2.0","id":"1","method":"listkeys"}' http://127.0
...
+      "keyState": 0,
+      "label": "test",
+      "p2wpkhScript": "0 952bae5f20dc6d0ca8a5b92b3aeafef46ac7c52c",
+      "pubkey": "038131d9917592ef142bad9db15d2d96ff4562f83105a50abbcc53362b8e1dffa6",
+      "pubKeyHash": "952bae5f20dc6d0ca8a5b92b3aeafef46ac7c52c",
+      "address": "tb1qj546uheqm3kse299hy4n46h7734v03fvdpz3wv"
     },
     {
-      "fullKeyPath": "84'/0'/0'/1/2",
+      "fullKeyPath": "86'/1'/0'/1/0",
```
Comment (lontivero) [inline docs/using-wasabi/RPC.md:497]: I think mainnet examples are better.

**Review by lontivero — CHANGES_REQUESTED** [review]
The examples are not okay IMO and we should extend the RPC to understand taproot keys too (and break backward compatibility before it is too late).

**Comment by MarnixCroes** [comment]
closing the PR (for now), until this RPC call is working properly.
I'll address your comments, thanks @lontivero

**Pull request closed without merging**